home/management/commands/setup_social_auth.py

```
@@ -0,0 +1,56 @@
+Management command to set up Google OAuth authentication
+"""
+import os
+from django.core.management.base import BaseCommand
+from django.contrib.sites.models import Site
+from allauth.socialaccount.models import SocialApp
+
+
+class Command(BaseCommand):
+    help = 'Set up Google OAuth authentication from environment variables'
+
+    def handle(self, *args, **options):
```

Ruff (0.14.4)

Unused method argument: `args`

(ARG002)